#!/usr/bin/env python
PACKAGE = "Modelos_Cinematicos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("base_arm1_joint",    double_t,    0, "An double parameter", 0,  -2.617, 2.617)
gen.add("arm1_arm2_joint",    double_t,    0, "An double parameter", 0,  -1.57,1.57)
gen.add("arm2_arm3_joint",    double_t,    0, "An double parameter", 0,  -2.09, 2.09)
gen.add("arm3_arm4_joint",    double_t,    0, "An double parameter", 0,  -2.617, 2.617)
gen.add("arm4_clamp1_joint",    double_t,    0, "An double parameter", 0.0,  -2.1, 1.57)
gen.add("clamp_joint",    double_t,    0, "An double parameter", 0,  0, 1.5)
gen.add("Simulacion", bool_t , 0, "Marcar para habilitar movimiento en simulaci√≤n",False)
gen.add("Dynamixel", bool_t , 0, "Marcar para habilitar movimiento robot real",False)
gen.add("Posicion_inicial", bool_t , 0, "Marcar para habilitar movimiento robot real",False)

exit(gen.generate(PACKAGE, "dynamic_angle_joint_node", "Angle_joint"))
